### ISSUE TITLE
lrzip-next -i should work even if file is encrypted

### ISSUE THREAD

**Issue by pete4abw** [pete4abw/lrzip-next#26]
While there are some difficulties currently in dealing with piped encrypted lrz files, there should be no reason why the command

`lrzip-next -i file.lrz`

should not work, even if the file is encrypted.

Asking for the password, or accepting it on the command line, `-efoo` or `--encrypt foo`, should work equally well as a decryption.

Something to look at...

**Comment by pete4abw** [comment]
Early tests for a 1 chunk small compressed file are promising: Note that items like file size, chunk size, chunk byte width are never stored with encrypted files. Each encrypted block is chained until an EOF marker is found. Since total file size is not stored, all compression ratios are meaningless. I may decide just to add a message about how these items can't be computed. But, it's a start. I'll push a branch on this after a little more work.
```
$ ./lrzip-next -vvi  ../../*git.tar.lrz
Using configuration file /home/peter/.lrzip/lrzip.conf
Detected lrzip version 0.7 file.
Encryption hash loops 4816896
Enter passphrase: 
Hashing passphrase 4816896 (16225333) times 
Rzip chunk 1:
Chunk byte width: 0
Chunk size: 0
Decrypting data        
Stream: 0
Offset: 26
Block   Comp    Percent Size
Decrypting data        
1       lzma    50.9%   97130 / 190876  Offset: 92      Head: 0
Decrypting data        
Stream: 1
Offset: 26
Block   Comp    Percent Size
Decrypting data        
1       lzma    78.8%   6015834 / 7633616       Offset: 97263   Head: 0
Rzip compression: 782449200.0% 7824492 / 0
Back end compression: 78.1% 6112964 / 7824492
Overall compression: 611296400.0% 6112964 / 0
../../lrzip-next.git.tar.lrz:
lrzip version: 0.7 file
Compression: rzip + lzma -- lc = 3, lp = 0, pb = 2, Dictionary Size = 33554432
Decompressed file size: 0
Compressed file size: 6113154
Compression ratio: 0.000*****
MD5 used for integrity testing
MD5: 5bc5ec0077c33fe9bf12ab9c3c0ea856
```

**Comment by pete4abw** [comment]
An in-progress commit has been made to branch **decrypt_info**. It's been tested with one chunk files, but not larger. Some fudging with the `offs`  variable before going to `next_chunk` may be needed in order to properly read the salt for the next encrypted block. Output has also been fixed to improve messages when the expected size of the decompressed file is not known (happens when output to STDOUT or if file is encrypted). Compare with above.

```
Due to using STDOUT or ENCRYPTION, expected decompression size not available
Rzip compression: Unavailable
Back end compression: 78.1% 6112964 / 7824492
Overall compression: Unavailable
../lrzip-next.git.tar.lrz:
lrzip version: 0.7 file
Compression: rzip + lzma -- lc = 3, lp = 0, pb = 2, Dictionary Size = 33554432
Decompressed file size: Unavailable
Compressed file size: 6113154
Compression ratio: Unavailable
MD5 used for integrity testing
MD5: 5bc5ec0077c33fe9bf12ab9c3c0ea856
```

**Comment by pete4abw** [comment]
Multi-Chunk encrypted files can have INFO shown. Also fixed inability to show percentages for encrypted files or those whose decompressed size is not known. i.e. written to STDOUT or ENCRYPTED.
```
Rzip chunk 2:
Chunk byte width: 0
Chunk size: 0
Stream: 0
Offset: 4696976595
Block   Comp    Percent Size
1       lzo     64.3%   1103054 / 1714931       Offset: 7054662282      Head: 0 Encrypted
Stream: 1
Offset: 4696976595
Block   Comp    Percent Size
1       lzo     23.0%   70471633 / 306668279    Offset: 4696976661      Head: 70471740  Encrypted
2       lzo     39.6%   121582588 / 306668279   Offset: 4767448335      Head: 192054369 Encrypted
3       lzo     48.1%   147572420 / 306668279   Offset: 4889030964      Head: 339626830 Encrypted
4       lzo     45.2%   138497664 / 306668279   Offset: 5036603425      Head: 478124535 Encrypted
5       lzo     43.4%   132997656 / 306668279   Offset: 5175101130      Head: 611122232 Encrypted
6       lzo     53.3%   163313785 / 306668279   Offset: 5308098827      Head: 774436058 Encrypted
7       lzo     40.5%   124211994 / 306668279   Offset: 5471412653      Head: 898648093 Encrypted
...
Due to using STDOUT or ENCRYPTION, expected decompression size not available
Rzip compression: Unavailable
Back end compression: 46.3% 7069596696 / 15253879930
Overall compression: Unavailable
filename.tar.lrz:
lrzip version: 0.7 file
Compression: rzip + lzo
Decompressed file size: Unavailable
Compressed file size: 7069599045
Compression ratio: Unavailable
MD5 used for integrity testing
MD5: f441f4144cc11ebe4a1f87b0542847cd
```

**Comment by pete4abw** [comment]
Fixed in master, 0.7.60
